htdocs/inc/functions.inc.php

```diff
@@ -372,6 +372,22 @@ function nf($number,$colorize=0,$ex=0)	// Number format
 		return $n;
 	}
 
+	/**
+	* Format number (round up)
+	*/
+	function nf_up($number,$colorize=0)	// Number format
```

Comment:
Instead of duplicating the code of the `nf` function can you either
```
function nf_up($number,$colorize=0)
{
    return nf(ceil($number), $colorize);
}
```
or call nf with `ceil($number)` directly

Reply:
Right. I prefer the first one and i also added missing nf_up's.